### DISCUSSION TITLE
how to put two textures to mobs

### DISCUSSION THREAD

**Discussion by CarL3106** [bridge-core/bridge.#392] [Q&A]
I would like to know how you can put two textures to a single peat on the bridge.

**Comment by Chikorita-Lover** [comment]
Could you be more specific? How are the two textures used?
- Used as variants? (similar to llamas, axolotls, etc.)
- Layered on each other? (similar to villagers)
- Swapped between under other circumstances? (similar to striders, wolves, etc.)
- Something else?
